File: ADS-2-Semestre/EngenhariaDeSoftware/InterfacePatoCase/src/pato/Voaveis_Asa.java
package pato;


public class Voaveis_Asa implements Padrao_Voaveis {

	private double velocidade;

	public Voaveis_Asa() {
		
	}

	public String voar() {
		return "Voando como um pássaro que voa. Velocidade: "
				+ getVelocidade();
	}

	public double getVelocidade() {
		return velocidade = 10;
	}

}

File: ADS-2-Semestre/EngenhariaDeSoftware/InterfacePatoC#/Pato/VoaveisAsa.cs
using Pato;
using System;
using System.Collections.Generic;
using System.Text;

namespace JogoPato
{
    public class VoaveisAsa : IPadraoVoaveis
    {
		private double velocidade;

		public VoaveisAsa()
		{
			velocidade = 10;
		}

		public String voar()
		{
			return "Voando. Velocidade: "
					+ getVelocidade();
		}

		public double getVelocidade()
		{
			return velocidade;
		}
	}
}
